packages/phone-number-privacy/combiner/src/pnp/services/thresholdState.ts

```diff
@@ -1,20 +1,21 @@
 import {
   PnpQuotaRequest,
-  PnpQuotaResponseSuccess,
+  PnpState,
+  SignMessageRequest,
   WarningMessage,
 } from '@celo/phone-number-privacy-common'
 import { Session } from '../../common/session'
 import { OdisConfig } from '../../config'
 
 // TODO(2.0.0, testing): add unit tests for this and domains equivalent
 // (https://github.com/celo-org/celo-monorepo/issues/9792)
-export class CombinerThresholdStateService<R extends PnpQuotaRequest> {
+export class CombinerThresholdStateService<R extends PnpQuotaRequest | SignMessageRequest> {
```

Comment:
@alecps how would you feel about me renaming this to `PnpThresholdStateService`? It feels weird to me that this is `CombinerThresholdStateService` given that `DomainThresholdStateService` is also a thresholdStateService in the combiner?

Reply:
Yea 100% agree